hstracker/appcast2.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?><rss xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle" version="2.0">
 <channel><item>
+    <title>1.7.1</title>
+    <pubDate>2020-10-15T20:17:28+00:00</pubDate><description>&lt;h2&gt;New:&lt;/h2&gt;
+&lt;ul&gt;
+&lt;li&gt;Updated for patch 18.4.2&lt;/li&gt;
+&lt;/ul&gt;</description>
+    <sparkle:minimumSystemVersion>10.12</sparkle:minimumSystemVersion>
+    <enclosure length="70233523" sparkle:dsaSignature="MCwCFB2Nl7FMKpTL7gyxLrEaVbzHvIOMAhRykNqrGBw4By3pxEERvlOnC1EXTQ==" sparkle:shortVersionString="1.7.0" sparkle:version="2146" type="application/octet-stream" url="https://github.com/HearthSim/HSTracker/releases/download/1.7.0/HSTracker.app.zip"/>
```

Comment:
I think this should say `sparkle:shortVersionString="1.7.0"` and `url="https://github.com/HearthSim/HSTracker/releases/download/1.7.0/HSTracker.app.zip"`?

Reply:
I've seen this before, this is why I added this line: https://github.com/HearthSim/HSTracker/blob/master/scripts/release.kts#L208

You might want to `rm -rf ~/Library/Caches/Sparkle_generate_appcast/`